DuplicateFinderEngine/Data/VideoFileEntry.cs

```diff
@@ -26,6 +43,11 @@ public VideoFileEntry(string file)
 		[ProtoMember(4)]
 		public MediaInfo mediaInfo;
 		[ProtoMember(5)]
-		public readonly bool IsImage;
-    }
+		public EntryFlags Flags;
```

Comment:
Can this break existing databases when you changed the type by preserving the protonmember field number?